chp9Lsn3/chp9Lsn3/NumberConversion.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chp9Lsn3
+{
+    public class NumberConversion
+    {
+        static void Main(string[] args)
+        {
+            int value = UserInput();
+            value = CaptureLastDigit(value);
+            string englishName = NameOfDigit(value);
+            PrintWord(englishName);
+
+        }
+
+        /// <summary>
+        /// Takes a value from the user.
+        /// </summary>
+        /// <returns>Integer</returns>
+        public static int UserInput()
+        {
+            int isInput = 0;
+            bool resolved = false;
+            Console.WriteLine("Please enter an integer.");
+
+            while (resolved == false)
+            {
+                string input = Console.ReadLine();
+                bool isInt = System.Int32.TryParse(input, out isInput);
+
+                if (isInt && isInput > 0)
+                {
+                    resolved = true;
+                }
+                else
+                {
+                    Console.WriteLine("Invalid Input!");
+                    resolved = false;
+                }
+            }
+
+            return isInput;
+        }
+
+        /// <summary>
+        /// Gets the remainder of the incoming integer when divided by ten.
+        /// </summary>
+        /// <param name="number">Integer</param>
+        /// <returns>Last digit of the number.</returns>
+        public static int CaptureLastDigit(int number)
+        {
+            number %= 10;
+            return number; 
+        }
+
+        /// <summary>
+        /// Gives the English name of the given number.
+        /// </summary>
+        /// <param name="digit">Single Digit</param>
+        /// <returns>String</returns>
+        public static string NameOfDigit(int digit)
+        {
+            string digitName = "Error";
+
+            switch (digit)
```

Comment:
Now that you've learned about arrays, it makes more sense to use an "array of strings" rather than a switch statement.